readme.md

```diff
@@ -337,6 +361,7 @@ The Kirby chart components use Highcharts. Note that this is a licensed product.
 [jest]: https://jestjs.io/
 [nrwl]: https://nrwl.io/
 [nx]: https://nx.dev/angular
+[roboto]: https://fonts.google.com/specimen/Roboto
```

Comment:
No longer relevant after importing from @fontsource
```suggestion
```

Reply:
should be resolved by 3d0f90a